### PR TITLE
[stable/cluster-autoscaler] Bump version to 1.13.7

### PR DESCRIPTION
**Pull request by coreypobrien**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 3.2.0
-appVersion: 1.13.1
+version: 3.3.0
```
Comment (yurrriq) [inline stable/cluster-autoscaler/Chart.yaml]: Thanks! Will you please rebase and bump this to `3.4.0`?

**Review by yurrriq — CHANGES_REQUESTED** [review]

File: stable/cluster-autoscaler/Chart.yaml (hunk 1 above)
Comment (coreypobrien) [reply stable/cluster-autoscaler/Chart.yaml]: 👍 Rebased.

**Review by yurrriq — APPROVED** [review]

**Pull request merged**